build/config/settings.json

```diff
@@ -8,5 +8,5 @@
   "coverage_folder": "tests/coverage",
   "asan_folder": "tests/coverage",
   "ubsan_folder": "tests/ubsan",
-  "ninja_launcher": "build/scripts/rexpy/rexpy/ninja_launcher.py"
+  "ninja_launcher": "build/scripts/rex-py/rexpy/ninja_launcher.py"
```

Comment:
naming guideline I follow is that folders don't have '-' in there.
where did this come from?

Reply:
RexStd was called rex_std, RexPy was called rexpy and glad & glm where called rex-glad and rex-glm I just picked one

Reply:
If files are not allowed to have no spaces I think folders should follow a similar guideline and I quite like the "-" between rex and the name of the library

Reply:
spaces are not recommened, I stuck with '_' because the source also has underscores

Reply:
I will change it to an "_"